### DISCUSSION TITLE
Samplers Missing

### DISCUSSION THREAD

**Discussion by BuildBackBuehler** [ssitu/ComfyUI_restart_sampling#12] [General]
Hi, I've been attempting to use this but the lack of a few samplers has me wondering

1. Does this methodology/concept breakdown with some of the samplers? I figure that's the issue at hand. But it'd be nice to use the KSampler node you provide in any case. As it stands right now, I'll likely end up injecting my go-to, the Efficient KSampler node, with the necessary restart-sampling inputs and have some if/only if statement for when to incorporate the restart-sampling segment input data. 
2. I imagine that no matter what, it would "work" to do the restart with any sampler – caveat lies with figuring out what restart-sampling segment input data makes it work with X/Y/Z samplers. But there's only so much time in a day so I guess I'd just have to wait until someone is up to that task.

**Comment by ssitu** [comment]
Are you referring to the SDE samplers being missing? I left them out because they don’t work with this technique. Unless I misread the paper, it boils down to whether the sampler can follow the sampling schedule one step at a time (aka sampling for 20 steps can be done using 20 individual ksampler nodes), but the SDE samplers keep track of state for the whole sampling schedule, so if it tries to do one step at a time it won’t work. There’s a portion of code at the top of `nodes.py` that holds a list of samplers to remove that you can remove entries from if you want to try them, but they don’t produce images when I tried them. I also left out unipc since it also didn’t work, not sure why.

**Comment by ssitu** [comment]
You can now try any sampler you want and see if the results are satisfactory using the new custom restart sampler node by @blepping.
